matrix_client/client.py

```diff
@@ -183,6 +204,27 @@ def login_with_password(self, username, password, limit=10):
         self._sync()
         return token
 
+    def login_with_jwt(self, jwt, limit=10):
+        """ Login to the homeserver using a JSON Web Token.
+
+        Args:
+            token (str): JWT
+            limit (int): Deprecated. How many messages to return when syncing.
+                This will be replaced by a filter API in a later release.
+
+        Returns:
+            str: Access token
+
+        Raises:
+            MatrixRequestError
+        """
+        token = self.login_with_jwt_no_sync(jwt)
+
+        """ Limit Filter """
```

Comment:
Please use actual python comments with `#` for comments rather than
copying bad style from elsewhere.